botorch/posteriors/deterministic.py

```diff
@@ -22,6 +22,10 @@ class DeterministicPosterior(Posterior):
     r"""Deterministic posterior."""
 
     def __init__(self, values: Tensor) -> None:
+        """
+        Args:
+            values: Values to deterministically sample from.
```

Comment:
"deterministically sampling" sounds a bit weird

```suggestion
        r"""
        Args:
            values: Values of the samples produced by this posterior.
```